.github/workflows/crypto-test-harness.yml

```diff
@@ -43,3 +43,40 @@ jobs:
         with:
           name: tcrypt_job
           path: ./tcrypt.out
+  cfgcheck:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y bc libelf-dev pahole
+          mkdir -p out
+      - name: Checkout
+        uses: actions/checkout@v3
```

Comment:
I think we're going to need to also update these to 4.2.0?
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsuses

In the build output, we have some notifications about node-16 EOL. So changing these will be their own commit ofc.